dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/fat/src/com/ibm/ws/jaxws/fat/FATSuite.java

```diff
@@ -31,6 +31,7 @@
                 HttpConduitPropertiesTest.class,
                 LoggingTest.class,
                 MTOMTest.class,
+                PropertySettingTest.class,
```

Comment:
Can we change the name to be a bit more specific? Something along the lines of `LibertyCXFPropertietTest`?